packages/blade/src/storybook-recipes/TopNavDashboard.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta } from '@storybook/react';
+import { RecipeSandbox } from '~utils/storybook/Sandbox/SandpackEditor';
+import parameters from '~utils/storybook/recipeParameters';
+
+export function TopNavDashboard(): JSX.Element {
+  return (
+    <RecipeSandbox
```

Comment:
is this still working? we were planning to remove these at some point so should we not add it?